engines/native/resultset.go

```diff
@@ -140,17 +140,21 @@ func (r *resultSet) ExtractFolder(path string, handler engines.FileHandler) erro
 }
 
 func (r *resultSet) Dispose() error {
-	// Halt all other sub-processes
-	err := system.KillByOwner(r.user)
-	if err != nil {
-		r.log.Error("Failed to kill all processes by owner, error: ", err)
-	}
+	var err error
+
+	if r.engine.config.CreateUser {
```

Comment:
We need an `else` part to this right? killing the process tree if we didn't use `CreateUser`.